docs/namespaces.md

```diff
@@ -74,4 +74,8 @@ will all use the default namespace, which is simply named "`default`".
 ```
 
 You can also use the `CONTAINERD_NAMESPACE` environment variable to specify the default namespace to use for
-any of the `ctr` client commands.
+any of the `ctr` client commands.  
+>To know more about different Namespaces packages click below links :  
+>[`Containerd Namespaces`](https://pkg.go.dev/github.com/containerd/containerd/v2/pkg/namespaces).  
+>[`kubernetes Namespaces`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).  
+>[`Moby(Docker) Namespaces`](https://docs.docker.com/engine/security/userns-remap/).
```

Comment:
No such concept exists on Moby (Docker).

User namespaces are the kernel's concepts.

Reply:
See https://man7.org/linux/man-pages/man7/namespaces.7.html for the kernel's namespaces

```
   Namespace types
       The following table shows the namespace types available on Linux.
       The second column of the table shows the flag value that is used
       to specify the namespace type in various APIs.  The third column
       identifies the manual page that provides details on the namespace
       type.  The last column is a summary of the resources that are
       isolated by the namespace type.
       Namespace Flag            Page                  Isolates
       Cgroup    CLONE_NEWCGROUP cgroup_namespaces(7)  Cgroup root
                                                       directory
       IPC       CLONE_NEWIPC    ipc_namespaces(7)     System V IPC,
                                                       POSIX message
                                                       queues
       Network   CLONE_NEWNET    network_namespaces(7) Network
                                                       devices,
                                                       stacks, ports,
                                                       etc.
       Mount     CLONE_NEWNS     mount_namespaces(7)   Mount points
       PID       CLONE_NEWPID    pid_namespaces(7)     Process IDs
       Time      CLONE_NEWTIME   time_namespaces(7)    Boot and
                                                       monotonic
                                                       clocks
       User      CLONE_NEWUSER   user_namespaces(7)    User and group
                                                       IDs
       UTS       CLONE_NEWUTS    uts_namespaces(7)     Hostname and
                                                       NIS domain
                                                       name
```

Reply:
I was less aware of it, thank you for your help